lib/pp.ml

```diff
@@ -201,11 +201,7 @@ let pp_with ft pp =
                                  pp_cmd s;
                                  pp_close_tag ft () [@warning "-3"]
   in
-  try pp_cmd pp
-  with reraise ->
-    let reraise = Exninfo.capture reraise in
-    let () = Format.pp_print_flush ft () in
```

Comment:
Isn't there a reason for this flush?

Reply:
Not that I can see, other than workarounding bugs in the caller.

Flushing is the responsibility of the creator for the formatter `ft`.